src/components/layout/footer/disclaimer-academy.js

```diff
@@ -118,7 +118,7 @@ const DisclaimerSectionAcademy = () => {
                             <DisclaimerParagraph>
                                 <Localize
                                     translate_text="CFDs are complex instruments and come with a high risk of losing money rapidly due to leverage. {{loss_percent}}% of retail investor accounts lose money when trading CFDs with this provider. You should consider whether you understand how CFDs work and whether you can afford to take the high risk of losing your money."
-                                    values={{ loss_percent: 66 }}
+                                    values={{ loss_percent: 62 }}
```

Comment:
Can we put this value in `constants.js`? so next time we need to change this, we don't need to go over the files again and we can just update it in one place.